docs/index.md

```diff
@@ -376,6 +376,120 @@ None
 
 ---
 
+## list.blobs
```

Comment:
Can we document these more? The current documentation doesn't really explain what these APIs are for. They seem redundant to other APIs we already have (e.g., `restic.list.snapshots` and `restic.snapshots`, `restic.list.keys` and `restic.key.list`).

Is there a real need for any of these? The goal of the repo is to provide useful Python bindings for interacting with restic, but I think the restic CLI is a bit too bloated for there to be value in providing wrappers for every CLI option.

Reply:
I've add a short explanation per method. Yes, snapshots/keys are redundant here. It feels inappropriate to leave some of the commands out just because they can be invoked differently as well. The user might not be aware that there are other ways and might be irritated why only a subset of function have been implemented.

Yes, I've had the need for `list locks`. I thought it's good practice/UX to add all others as well because the work wasn't that much more.

Reply:
I don't want to support all the CLI commands, just the ones that are useful. Even if they're easy to implement, there's a cost to maintaining them over time.

Can we reduce this PR to just `restic.list.locks`?